docs/migration-guide.md

```diff
@@ -2,11 +2,196 @@
 
 ## Contents
 
+- [Neutrino v8 to v9](#neutrino-v8-to-v9)
 - [Neutrino v7 to v8](#neutrino-v7-to-v8)
 - [Neutrino v6 to v7](#neutrino-v6-to-v7)
 - [Neutrino v5 to v6](#neutrino-v5-to-v6)
 - [Neutrino v4 to v5](#neutrino-v4-to-v5)
 
+## Neutrino v8 to v9
+
+Neutrino v9 is our largest release ever, bringing the preset and middleware ecosystem back to the
+native tools and utilities for which they were originally created. The biggest breaking change
+is the necessity to use external tools *alongside* Neutrino now, i.e. webpack, ESLint, Jest, Karma,
+and others' native CLIs will be used in tandem with Neutrino. A list of changes is detailed below for
+migrating:
+
+- **BREAKING CHANGE** The minimum supported Node.js version is v8.3. Node.js v9 is no longer supported
```

Comment:
`s/8.3/8.10/` (#1035)

Also, I think Node.js has dropped the `v` prefix? (Due to confusion with `v8` etc)

Reply:
Weird, never heard this, although I thought the big differentiator should have been the casing.

Reply:
> Note that, when referring to Node.js release versions, we have dropped the "v" in Node.js 8. Previous versions were commonly referred to as v0.10, v0.12, v4, v6, etc. In order to avoid confusion with V8, the underlying JavaScript engine, we've dropped the "v" and call it Node.js 8.

From https://nodejs.org/en/blog/release/v8.0.0/#long-term-support

(I had to look it up to check I wasn't mis-remembering! :-))

Reply:
TIL!